src/components/Image/src/Image.vue

```diff
@@ -81,7 +91,7 @@ export default {
 		shape: {
 			type: String,
 			default: undefined,
-			validator: (shape) => ['squared', 'rounded', 'pill'].includes(shape),
```

Comment:
renaming `squared` to `square` and removing `rounded` and `pill` counts as a breaking change (even if nobody was using them), we should update the PR title from `feat(image):` to `feat(image)!:`